### ISSUE TITLE
Question about “test_ELCFS.py”.

### ISSUE THREAD

**Issue by zzzqzhou** [liuquande/FedDG-ELCFS#5]
Dear author,
In code file “test_ELCFS.py”  line 63,  you used "test_net.train()" in test stage. However, as we know, we typically use "model.eval()" operation during the test stage. Is there any reason for you to do so?

**Comment by liuquande** [comment]
Hi zzzqzhou,

We use model.train() instead of model.eval() since we expect to use the statistics of testing data (in BN layer) for feature normalization during inference. 
Given the domain shift between training and testing data, this can yield a better generalization performance compred with using the statistics from training sets.

**Comment by zzzqzhou** [comment]
I see. Thanks for your reply. Regards.
